test/spix/smoke_test.py

```diff
@@ -26,18 +26,36 @@ def test_start_app(app, screenshot_path, extra, process_alive):
     """
     Starts a test app to the welcome screen and creates a screenshot.
     """
+    if not os.environ.get("SPIX_PROJECT_FILE") is None:
```

Comment:
- This should be added as a new test, next to `test_start_app`
- The filename should be coded in this file (not in an env variable)
- Ideally, we would even [parameterize the test](https://github.com/opengisch/qgis_server_render_geojson/blob/master/tests/test_request.py#L54-L67), so we can run the same test on a matrix of files

Reply:
@m-kuhn , +1 to make a new test_message_logs test, that makes a lot of sense.

As for the filename not being an environment variable, I'm open to suggestions but I'm not sure how we'd get the source tree path to direct us to the test/testdata/...qgz to begin with. Also, we'd need to do it in the other file (conftest.py) as we need for it to be loaded via exec argument. Thoughts on ways to proceed are most welcome :)

Reply:
The path to this python file could help?
Otherwise we could also just add a `TESTDATA` path as env var.

Reply:
@m-kuhn , path to python file sounds good.

Reply:
@m-kuhn , I've pushed something, let me know what you think of it.